fronts-client/src/components/FrontsEdit/Collection.tsx

```diff
@@ -178,6 +178,7 @@ class CollectionContext extends React.Component<ConnectedCollectionContextProps>
                         getNodeProps={() => getAfNodeProps(isUneditable)}
                         onSelect={() => selectCard(card.uuid, id, false)}
                         onDelete={() => removeCard(group.uuid, card.uuid)}
+                        groupSizeId={group.id ? parseInt(group.id) : 0}
```

Comment:
I see this value is drilled down to `ArticleFormMeta` and used to determine `allowGigaBoost` - but its not clear here what this value signifies, or why it defaults to 0 - would it work to replace this prop with a boolean with a name like `groupSupportsGigaboost`, `groupSizePermitsGigaboost` or similar?